conda/recipes/nvstrings/build.sh

```diff
@@ -1,11 +1,20 @@
 #!/usr/bin/env bash
-cd python
-$PYTHON setup.py install
-mkdir -p $PREFIX/include/nvstrings
-mkdir -p $PREFIX/lib
-cp ../cpp/include/NVStrings.h $PREFIX/include/nvstrings
-cp ../cpp/include/NVCategory.h $PREFIX/include/nvstrings
-cp ../cpp/include/NVText.h $PREFIX/include/nvstrings
-cp build/lib.linux-x86_64*/libNVStrings.so $PREFIX/lib
-cp build/lib.linux-x86_64*/libNVCategory.so $PREFIX/lib
-cp build/lib.linux-x86_64*/libNVText.so $PREFIX/lib
+
+CMAKE_COMMON_VARIABLES=" -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_BUILD_TYPE=Release"
+
+# show environment
+printenv
+# Cleanup local git
+git clean -xdf
+# Change directory for build process
+cd cpp
+# Use CMake-based build procedure
+mkdir build
+cd build
+# configure
+cmake $CMAKE_COMMON_VARIABLES ..
+# build/install cpp
+make -j${PARALLEL_LEVEL} VERBOSE=1 install
+# build/install python
+cd ../../python
+$PYTHON setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
@jakirkham I tried using `pip install` here but it wouldn't move the actual `*.py` files into the conda package. I imagine I have a dependency issue where it's grabbing the wrong pip and installing into the wrong location because of that.

Reply:
So one trick that comes in handy is `$PYTHON -m pip`. This ensures the `pip` installed in that Python environment is used.

FWIW leaving things as-is is fine from my perspective.